libs/RobotKit/include/RobotController.h

```diff
@@ -482,6 +482,11 @@ class RobotController : public interface::RobotController
 		auto on_charge_did_stop = [this]() { raise(event::charge_did_stop {}); };
 		_battery.onChargeDidStop(on_charge_did_stop);
 
+		_service_monitoring.registerGoToSleepRequested([this] {
+			raise(event::go_to_sleep_requested {});
```

Comment:
If I understand correctly, this will trigger the sleeping animation with the LEDs and video. is that correct?